book/content/continuous_integration/continuous_integration.md

```diff
@@ -86,11 +87,22 @@ Technically speaking the above explanation conflates three related concepts, con
 
 In this chapter this entire process is referred to as continuous integration for the sake of simplicity.
 
+<a name="What_options_are_available"></a>
+### What are the options for CI service providers?
+
```

Comment:
> such as Travis, Circle CI, GitHub Actions, GitLab CI

This list should (IMO) either be just a couple or *all* of the ones you are going to describe below.

Reply:
Good point -- I went for just a couple, to keep it short.